releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -2,6 +2,50 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0 - September 24, 2024
+
+### Enhancements
+
+#### Variants
+
+```csharp
+Variant variant = await featureManager.GetVariantAsync(MyFeatureFlags.HelpText);
+
+model.Text = variant.Configuration.Value;
+```
+
+Variants are a tool that can be used to surface different variations of a feature to different segments of an audience. Previously, this library only worked with flags. The flags were limited to boolean values, as they are either enabled or disabled. Variants have dynamic values. They can be any valid json value- like a string, int, or complex object.
```

Comment:
This sounded like variants are not feature flags anymore. Variants are an enhancement to feature flags. Feature flags with variants are still flags.

Reply:
Hmm. I agree. This was taken from the `preview1` release notes but it does sound a little incorrect. How about:

```
This library now supports feature flags with variants. Variants are a tool that can be used to surface different variations of a feature to different segments of an audience. Previously, flags were limited to boolean values, as they are either enabled or disabled. Variants have dynamic values. There can be more than two and they can be any valid json value- like a string, int, or complex object.
```